examples/AccountManagement/GetAccountHierarchy.php

```diff
@@ -0,0 +1,196 @@
+            . ' customer_client.id'
+            . ' FROM customer_client';
+
+        // Performs a breadth-first search algorithm to build an associative array mapping managers
+        // to their child accounts ($customerIdsToChildAccounts).
+        $unprocessedManagerAccounts = [$customerId];
+        $customerIdsToChildAccounts = [];
+        $rootCustomerClient = null;
+        while (!empty($unprocessedManagerAccounts)) {
+            $managerAccountId = array_shift($unprocessedManagerAccounts);
+            // Issues a search request by specifying page size.
+            $response = $googleAdsServiceClient->search(
+                $managerAccountId,
+                $query,
+                ['pageSize' => self::PAGE_SIZE]
+            );
+
+            // Iterates over all rows in all pages to get all customer clients under the specified
+            // customer's hierarchy.
+            foreach ($response->iterateAllElements() as $googleAdsRow) {
+                /** @var GoogleAdsRow $googleAdsRow */
+                $customerClient = $googleAdsRow->getCustomerClient();
+                // The customer client that has level of 0 is the specified customer.
+                if ($customerClient->getLevelUnwrapped() === 0) {
+                    // Store the root customer client, which is the first encountered customer
+                    // client that has level of 0.
+                    if (is_null($rootCustomerClient)) {
+                        $rootCustomerClient = $customerClient;
+                    }
+                    continue;
+                }
+
+                // For all level-1 (direct child) accounts that are a manager account, the above
+                // query will be run against them to create an associative array of managers to
+                // their child accounts for printing the hierarchy afterwards.
+                $customerIdsToChildAccounts[$managerAccountId][] = $customerClient;
+                if ($customerClient->getManagerUnwrapped()) {
+                    // A customer can be managed by multiple managers, so to prevent visiting the
+                    // same customer many times, we need to check if it's already in the map.
+                    $alreadyVisited = array_key_exists(
+                        $customerClient->getIdUnwrapped(),
+                        $customerIdsToChildAccounts
+                    );
+                    if (!$alreadyVisited && $customerClient->getLevelUnwrapped() === 1) {
+                        array_push($unprocessedManagerAccounts, $customerClient->getIdUnwrapped());
+                    }
+                }
+            }
+        };
+
+        printf(
+            "The hiearchy of the customer ID %d is printed below:%s",
+            $rootCustomerClient->getIdUnwrapped(),
+            PHP_EOL
+        );
+        print 'Customer ID (Descriptive Name, Currency Code, Time Zone)' . PHP_EOL;
```

Comment:
nit: The format specified here is really dictated by the impl of `printAccountHierarchy()`. I'd move it there under a conditional of `if $depth === 0`